e2e/cypress/integration/interactive_menu/basic_options.js

```diff
@@ -106,7 +108,7 @@ describe('MM-15887 Interactive menus - basic options', () => {
             cy.get('.select-suggestion-container > input').should('be.visible').and('have.attr', 'value', options[0].text);
         });
 
-        cy.wait(TIMEOUTS.TINY);
+        cy.wait(TIMEOUTS.SMALL);
```

Comment:
I increased the `wait` here to ensure that it'll assert the ephemeral post after clicking the option.  Can't think of a way to better handle it.

Reply:
Yeah, that's tricky. I feel like we have the same issue for knowing when a new post arrives. We could do something like track the last post ID, and wait for a new one, but that feels like a lot of overheard vs  "hey, just wait a second".